scripts-available/CDB_Overviews.sql

```diff
@@ -35,15 +35,28 @@ AS $$
   DECLARE
     schema_name TEXT;
     base_table_name TEXT;
+    disable_overviews BOOLEAN;
   BEGIN
-    SELECT * FROM _cdb_split_table_name(reloid) INTO schema_name, base_table_name;
-    RETURN QUERY SELECT
-      reloid AS base_table,
-      _CDB_OverviewTableZ(table_name) AS z,
-      table_regclass AS overview_table
-      FROM _CDB_UserTablesInSchema(schema_name)
-      WHERE _CDB_IsOverviewTableOf((SELECT relname FROM pg_class WHERE oid=reloid), table_name)
-      ORDER BY z;
+    SELECT COALESCE(value::text = 'true', false)
+    FROM cdb_conf WHERE key = 'disable_overviews'
+    INTO disable_overviews;
+
+    IF disable_overviews THEN
+      RETURN QUERY SELECT
+        NULL::regclass AS base_table,
+        NULL::integer AS z,
+        NULL::regclass AS overview_table
+      WHERE false;
```

Comment:
Interesting! Why do you use `WHERE false;` here? What's the result of omitting `WHERE false` altogether?

Reply:
`WHERE false;` ensures that 0 rows are returned, which is the desired result.  Omitting this clause would give 1 row with `NULL` values.

Reply:
Makes perfect sense.. 
select  NULL::regclass AS base_table,NULL::integer AS z,NULL::regclass AS overview_table where false;
 base_table | z | overview_table 
------------+---+----------------
(0 rows)
select  NULL::regclass AS base_table,NULL::integer AS z,NULL::regclass AS overview_table ;
 base_table | z | overview_table 
------------+---+----------------
            |   | 
(1 row)